Cargo.toml

```
@@ -183,3 +183,4 @@ encrypt-cache-results = ["dep:chacha20poly1305", "dep:chacha20"]
 cec-cache-results = ["compress-cache-results", "encrypt-cache-results"]
 experimental-io-uring = ["actix-web/experimental-io-uring"]
 use-non-static-synonyms-search = ["thesaurus"]
+socks = ["reqwest/socks"]
```

```suggestion
torified-search-results = ["reqwest/socks"]
```

I think we should rename this feature to something like `torified-search-results` because the name `socks` sounds quite misleading for the user of the search engine like it could sound like the search engine supports running behind `tor` which is as you know not the case. So a name like `torified-search-results` will better explain the feature. What do you think? 🙂

---

> because the name socks sounds quite misleading for the user of the search engine like it could sound like the search engine supports running behind tor which is as you know not the case. 
I have no idea what you're talking about. `socks` by itself has nothing to do with tor, and there are many other proxies/applications that make use of the protocol that have nothing to do with tor. And likewise there are other proxies that people may want to use other than tor.